docs/managing-compute-resources.asciidoc

```diff
@@ -1,15 +1,13 @@
 [id="{p}-managing-compute-resources"]
 == Managing compute resources
 
-When a Pod is created it may request CPU and RAM resources. It may also specify the maximum resources that the containers are allowed to consume. Both Pod `limits` and `requests` can be set in the specification of any object managed by the operator (Elasticsearch, Kibana or the APM server). For more information about how this is used by Kubernetes please see https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/[Managing Compute Resources for Containers].
+When you create a Pod, you must specify the requests in terms of CPU and RAM resources, and the limits in terms of maximum resources that containers can consume. Then, you can set both `requests` and `limits` in the specification of the object managed by the operator: Elasticsearch, Kibana or the APM server. For more information, see https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/[Managing Compute Resources for Containers].
```

Comment:
Pods are not created by the user, not sure that using "you" makes sense, also it's optional.